test/spmd/test_xla_spmd_python_api_interaction.py

```diff
@@ -54,6 +55,55 @@ def test_xla_replication_devices(self):
     self.assertEqual(xm.xla_real_devices([device]), [device_type + ':0'])
 
 
+class BasicRuntimeAPITest(test_xla_sharding_base.XlaShardingTest):
+
+  @classmethod
+  def setUpClass(cls):
+    os.environ["XLA_USE_SPMD"] = "1"
```

Comment:
Do we need to cover the `XLA_USE_SPMD=0` case, or is that covered by the existing tests?

Reply:
I assumme all Non-SPMD case is covered by original test.